pkg/logger/init.go

```diff
@@ -69,6 +69,7 @@ type Options struct {
 	// OutputFilePath is an absolute file path. The file will be created if it doesn't exist.
 	// All logs available at level 9 will be written to the file.
 	OutputFilePath string
+	ManagementClusterState string
```

Comment:
It doesn't seem like this is used by anything in the logger package - what's its purpose?

Reply:
The purpose of using the logger package is the flow is kind of similar to eksa-cli.log file we generate and delete after cluster creation succeeds and keep it if cluster creation fails. Also, we want this folder from many other packages like workflows, clustermanager, executables, etc so logger package seemed suitable for that.

Reply:
I appreciate the motivation, however logger output file path configuration and management cluster state snapshots are 2 very different concepts with the latter not really having anything to do with logging.

We want to inject the path into the data structures that require it as opposed to have them rely on global state.

Reply:
As discussed, moved ManagementClusterState to task package.